build_tools/circle/build_doc.sh

```diff
@@ -153,35 +153,12 @@ export PATH="/usr/lib/ccache:$MINICONDA_PATH/bin:$PATH"
 ccache -M 512M
 export CCACHE_COMPRESS=1
 
-# Old packages coming from the 'free' conda channel have been removed but we
-# are using them for our min-dependencies doc generation. See
-# https://www.anaconda.com/why-we-removed-the-free-channel-in-conda-4-7/ for
-# more details.
-if [[ "$CIRCLE_JOB" == "doc-min-dependencies" ]]; then
```

Comment:
This is from a while ago and only made sense when we were using the defaults channel (now using conda-forge) so I removed this